common/util/vector_tree.h

```diff
@@ -239,22 +407,18 @@ class VectorTree : private _VectorTreeImpl {
   typedef std::vector<this_type> subnodes_type;
```

Comment:
Shouldn't this be typedef'ed to`VectorTreeChildrenList` now ?

Reply:
Not really. It is used for:
- Getting iterator types - `VectorTreeChildrenList` would be OK for this.
- Creating some temporary container which is compatible with node's children list ([align_test.cc](https://github.com/chipsalliance/verible/blob/master/common/formatting/align_test.cc#L1492), [vector_tree.h](https://github.com/chipsalliance/verible/pull/1252/files#diff-46de95dbc32f853fc3fbec5c4e9e6e01c7906fe79b5e49006d447c13a5a23335R843-R844)) - `VectorTreeChildrenList` is either impossible to use (it can't be constructed outside of VectorTree) or adds overhead of parent pointer assigning when this is not needed.

I think in the next refactoring (conversion of VectorTree methods into functions operating on a template type fulfilling "Tree" concept) this typedef will be probably removed and replaced with use of `tree_traits` class.